crypto/core_namemap.c

```diff
@@ -508,16 +506,28 @@ OSSL_NAMEMAP *ossl_namemap_stored(OSSL_LIB_CTX *libctx)
     return namemap;
 }
 
-OSSL_NAMEMAP *ossl_namemap_new(void)
+OSSL_NAMEMAP *ossl_namemap_new(OSSL_LIB_CTX *libctx)
 {
     OSSL_NAMEMAP *namemap;
+    HT_CONFIG htconf = { NULL, NULL, NULL, NAMEMAP_HT_BUCKETS, 1, 1 };
```

Comment:
I think the first member of the configuration should be libctx here, no?  I assume thats why you modified the function parameter list.

Reply:
only the one nit, otherwise looks good

Reply:
It is set on line 514. AFAIK setting this directly in the initializer will break ANSI build.